Changes

```diff
@@ -38,8 +34,16 @@ Working version
   (Arthur Charguéraud and Armaël Guéneau, with help from Gabriel Scherer and
   Frédéric Bour)
 
+- GPR#1491: Improve error reporting for ill-typed applicative functor
+  types, F(M).t.
+  (Valentin Gatien-Baron)
+
 ### Code generation and optimizations:
 
+- GPR#1370: Fix code duplication in Cmmgen
+  (Vincent Laviron, with help from Pierre Chambart,
+   reviews by Gabriel Scherer and Luc Maranget)
+
```

Comment:
That shouldn't be here.

Reply:
I don't see why not. It's about the same as fixing a typo in a comment.

Reply:
I agree with @sliquister .
I had already noticed the issue [here](https://github.com/ocaml/ocaml/pull/1468#discussion_r149052600) but it's apparently not worth anyone's time to create a separate PR. So fixing it here sounds good.

Reply:
Ah, I though it was a change that was included by mistake. Not sure why it's included in a commit that has literally nothing to do with it, but fair enough. :)